components/related-items/README.md

```diff
@@ -7,12 +7,36 @@ RelatedItems
 ```
 <!-- STORY -->
 
+### Usage
 
+Simple
+```jsx
+import UnorderedList from '@govuk-react/unordered-list';
+import ListItem from '@govuk-react/list-item';
+import { asAnchor } from '@govuk-react/hoc';
+
+const AnchorTag = asAnchor('a');
+
+<RelatedItems>
+  <Header level={3}>Example header</Header>
```

Comment:
consider adding an import for Header as well.

```jsx
import Header from '@govuk-react/header';
```

Reply:
Spotted, nice one